### ISSUE TITLE
FOUC on All Interactive Elements

### ISSUE THREAD

**Issue by randycasburn** [jarek-foksa/xel#98]
**Given**:  

- Chrome: Version 80.0.3987.16 (Official Build) dev (64-bit)
- any on-line interactive widget demonstration @ https://xel-toolkit.org/elements
- any of the three themes

**When**: interacting with an interactive widget:

- accordion
- button
- tab
- doctab
- menu*
- contextmenu
- notification
- select
- colorselect

**Then**: At the end of the animation, dismissal, etc., there is an extra flash of the content. The most dramatic example of this is on the custom style dialog.

**Comment by jarek-foksa** [comment]
I'm unable to reproduce this issue on macOS with Chrome 81.0.4031.0 (current Canary version) or Chrome 79.0.3945.130 (current stable version).

What operating system are you using? Did you enable any experimental Chrome flags?

![2020-01-19 22 55 45](https://user-images.githubusercontent.com/193371/72689160-d9ed2600-3b0e-11ea-9c7f-0010c5371cee.gif)

**Comment by randycasburn** [comment]
**Resolved**!

I turned off the Experimental Web Platform chrome://flag and that fixed it.

---

For future reference here are answers to your questions and what I saw:

![flash](https://user-images.githubusercontent.com/1176839/72690489-87961000-3ae2-11ea-88f6-eb20dc3d2b8e.gif)

1. MacOS Catalina v10.152
2. Yes, Experimental Web Platform & Experimental JavaScript
3. Chrome stable: 
Google Chrome is up to date
Version 81.0.4029.3 (Official Build) dev (64-bit)
